server/src/main/java/org/opensearch/throttling/admissioncontrol/enums/AdmissionControlMode.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.throttling.admissioncontrol.enums;
+
+import java.util.Locale;
+
+/**
+ * Defines the AdmissionControlMode
+ */
+public enum AdmissionControlMode {
+    /**
+     * AdmissionController is completely disabled.
+     */
+    DISABLED("disabled"),
+
+    /**
+     * AdmissionController only monitors the rejection criteria for the requests.
+     */
+    MONITOR("monitor_only"),
+
+    /**
+     * AdmissionController monitors and rejects tasks that exceed resource usage thresholds.
+     */
+    ENFORCED("enforced");
```

Comment:
I am still think to have the monitor mode atleast in first phase of rollout and it will also help when we are trying to add new controller to track functionality without any impact on cluster.